app/celery/tasks.py

```diff
@@ -13,7 +14,9 @@ def publish_govuk_alerts(self, broadcast_event_id=""):
     try:
         alerts = Alerts.load()
         rendered_pages = get_rendered_pages(alerts)
-
+        if os.environ.get("HOST") == "local":
+            current_app.logger.info("Skipping upload to S3 in local environment")
+            return
```

Comment:
Would it make more sense to make this a config option in `config.py` pulled from the environment rather than directly checking the environment here?  Something like `DISABLE_S3_UPLOAD`

That said, it might make more sense to move the `GOVUK_ALERTS_S3_BUCKET_NAME` config option only into the Hosted config and check for the existence of that, if it's not found don't upload to S3.